fbbt/src/ontologies/fbbt-edit.obo

```diff
@@ -63575,6 +63576,95 @@ intersection_of: FBbt:00005106 ! neuron
 intersection_of: capable_of_part_of GO:0050954 ! sensory perception of mechanical stimulus
 relationship: part_of FBbt:00007687 ! mechanosensory system
 
+[Term]
+id: FBbt:00007701
+name: longitudinal subdivision of central nervous system
+def: "Any subdivision of the central nervous system along its anterior-posterior axis." [FB:dos]
+is_a: FBbt:00007060 ! multi-cell-component structure
+relationship: part_of FBbt:00005094 ! central nervous system
+property_value: created_by "dos" xsd:string
+creation_date: 2018-01-22T13:32:01Z
+
+[Term]
+id: FBbt:00007702
+name: longitudinal subdivision of ventral nerve cord
+def: "Any subdivision of the ventral nerve cord along its anterior-posterior axis." [FB:dos]
+intersection_of: FBbt:00007701 ! longitudinal subdivision of central nervous system
+intersection_of: part_of FBbt:00001102 ! ventral nerve cord
+property_value: created_by "dos" xsd:string
+creation_date: 2018-01-22T13:34:56Z
+
+[Term]
+id: FBbt:00007703
+name: centromedial subdivision of ventral nerve cord
+def: "A centro-medially located longitudinal subdivision of the larval ventral nerve cord spanning the gnathal, thoracic and abdominal segments to abdominal segment 8.  An axial plane just dorsal to the intermediate commissures and CITd (centro intermediate tract dorsal, `CI1` [FBbt_00005905]) specifies the dorsal boundary of this domain.  A virtual axial plane intersecting the ventral centro-intermediate tract (CITv) and the VMT (`Ventro Medial Tract` [FBbt_00001646]) forms the ventral boundary of this domain.  (CITv encompasses `CI2`[FBbt_00005906] and `CI3`[FBbt_00005907])." [FB:dos] {comment="PMID:28730682"}
+is_a: FBbt:00007701 ! longitudinal subdivision of central nervous system
+relationship: part_of FBbt:00001102 ! ventral nerve cord
+property_value: created_by "dos" xsd:string
+creation_date: 2018-01-22T13:38:56Z
+
+[Term]
+id: FBbt:00007704
+name: centrolateral subdivision of ventral nerve cord
+def: "A centro-laterally located longitudinal subdivision of the larval ventral nerve cord spanning the gnathal, thoracic and abdominal segments to abdominal segment 8.  An axial plane placed just dorsally of the intermediate commissures and CITd (centro intermediate tract dorsal, `CI1` [FBbt_00005905]) specifies the dorsal boundary of this domain.  A virtual axial plane intersecting the ventral centro-intermediate tract (CITv) and the VMT (`Ventro Medial Tract` [FBbt_00001646]) forms the ventral boundary of this domain. This is medial to the central domain. (CITv encompasses `CI2`[FBbt_00005906] and `CI3`[FBbt_00005907])." [FB:dos] {comment="PMID:28730682"}
+is_a: FBbt:00007701 ! longitudinal subdivision of central nervous system
+relationship: part_of FBbt:00001102 ! ventral nerve cord
+property_value: created_by "dos" xsd:string
+creation_date: 2018-01-22T14:04:04Z
+
+[Term]
```

Comment:
below OK but grammar is strange on end, add "defined at the" between "boundary" and "ventral"